src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -68,6 +79,17 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
             isResizable: true
         };
     });
+
+    const columns: IColumn[] =
+        additionalColumns.length > 0
+            ? staticColumns.concat(additionalColumns)
+            : staticColumns;
+    // mark each column based on whether it's currently the one sorted
+    columns.forEach((x) => {
+        x.isSorted = sortKey === x.fieldName;
+        x.isSortedDescending = isSortedDescending;
```

Comment:
we should not set the value to isSortDescending. It should be set to false unless the columnname matches the sort key, then use the state variable. This way when you click on one column, and then another column, it always starts in the same sort order instead of being dependent on the last column's sort direction. That is how most lists work so we should follow that behavior so users have a consistent and expected experience.